week2.Rmd

```diff
@@ -0,0 +1,130 @@
+---
+title: "Week 2 Problem Set"
+subtitle: "The Stefan-Boltzmann Law and Albedo"
+author: "Juliana Olsen-Valdez"
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  pdf_document:
+    toc: yes
+    toc_depth: '3'
+  html_document:
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: yes
+    toc_depth: 3
+    code_folding: show
+
+editor_options:
+  chunk_output_type: inline
+---
+> Click into a code cell (the gray blocks below) to select or edit it. To run a selected code cell, hit `Ctrl`+`Shift`+`Enter` (Windows & Linux) / `Command`+`Shift`+`Enter` (MacOS). Make sure that each code cell runs successfully before you move on to the next one.
+
+Load the libraries that we'll use in this Problem Set: 
+```{r load libraries}
+library(dplyr)
+library(ggplot2)
+```
+
+##Introduction
+Last week, we introduced the Stefan-Boltzmann Law and applied it to questions regarding astronomical energies and temperatures, specifically with respect to black bodies. This week, we'll build on the Stefan-Boltzmann Law to include terms like albedo and calculations related to climate. 
+
+##Learning Goals
+By the end of this exercise, you will be able to:
+1) Describe what albedo is, how it relates to emissivity, and how it can be substituted into the Stefan-Boltzmann Law equation. 
+2) Calculate the temperature of the earth given different values for albedo. 
+3) Use a basic plotting function and data to visualize quantitative values related to albedo. 
+
+##Background and Defining Terms
+Let's first re-familiarize ourselves with the Stefan-Boltzmann Law:
+$$ \begin{align} \frac {P}{A} = e \sigma T^4 \end{align}$$
+Terms and Units: 
+$P = Power\ (Watts)$  
```

Comment:
if you could put these in SI base units as well, that would be helpful! (Watts = kg m^2/s^3)